### PR TITLE
feat : added homePage 

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It contains :
* Header
* Hero-section
* Features 
* Categories
* Products
* Footer

### CODE REVIEW

**Review by neetukumari4858 — APPROVED** [review]
overall code is nicely written.It is good to merge.

File: index.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.button-shop-now{
+    display: block;
+}
+.brand{
+    width:200px;
+    height:100px;
+}
+.brand img{
+    width:150px;
+    height:100px;
```
Comment (t007rushi) [inline index.css:28]: rem???

File: common.css [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  }
+  
+  .flex-row{
+    display: flex;
+  }
+  .icon-white{
+    color:var(--light-color);
+  }
+  .icon-black{
+    color:var(--)
```
Comment (t007rushi) [inline common.css:54]: empty color?

File: common.css [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    
+  
+  }
+  body{
+    overflow-x: hidden;
+    max-width: 2000px;
```
Comment (t007rushi) [inline common.css:19]: rem?

**Review by t007rushi — CHANGES_REQUESTED** [review]
Overall UI Looks Clean

File: index.css (hunk 1 above)
Comment (Priya-Patel31) [reply index.css:28]: rems are used for font-size ,padding ,margin as they get affected but for width and height we can use px

**Review by t007rushi — APPROVED** [review]

**Pull request closed without merging**